src/routes/recordRouter.js

```diff
@@ -0,0 +1,15 @@
+const { throwError } = require("../utils");
+const express = require("express");
+const recordController = require("../controllers/recordController");
+
+const recordRouter = express.Router(); //라우터를 시작합니다.
+
+const createRecord = recordController.createRecord;
+const readRecord = recordController.readRecord;
+
+recordRouter.post("/createRecord", recordController.createRecord);
+recordRouter.get("/readRecord/:id", recordController.readRecord);
```

Comment:
```suggestion
recordRouter.post("/", recordController.createRecord);
recordRouter.get("/:id", recordController.readRecord);
```
- 정석적인 RESTful API 규칙에 따라 uri를 만들면, HTTP method만으로도 어떤 동작을 수행할 지 명확히 전달할 수 있기 때문에, (post, get) uri에 동사를 포함하지 않아도 됩니다

Reply:
반영했습니다!

Reply:
반영했습니다.